### PR TITLE
Update sales report to compatibility with refund update

### PR DESCRIPTION
**Pull request by Mrlunsuper**
### All Submissions:

* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

- Updated: View order screen in sales report table in Vendor Dashboard
- Updated: Orders table view items in Vendor Dashboard

### How to test the changes in this Pull Request:

1. Login as vendor account
2. Go to Vendor Dashboard
3. Click Show Orders in the Sales Report table will see refund detail if the order has been refunded
4. Click View items in the Orders table will see refund detail if the order has been refunded

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### CODE REVIEW

**Review by digitalchild — CHANGES_REQUESTED** [review]
Move the setting from Pro into marketplace and remove the Pro requirement.

File: templates/dashboard/orders.php [hunk 1]
```diff
@@ -162,21 +167,38 @@
...
 
+						if ( $refunded_total > 0 ) {
+							$refunded_qty = $is_full_refuned ? ( -1 * $item['qty'] ) : $order->get_qty_refunded_for_item( $item->get_id() );
+							$refunded[]   = $refunded_qty . ' x ' . wc_price( $refunded_total ) . ' x ' . $item['name'];
+						}
+						?>
 
 					<?php endforeach; ?>
+					<?php if ( ! empty( $refunded ) ) : ?>
+						<br/><strong><?php echo esc_html__( 'Refuned:', 'wc-vendors' ); ?></strong>
```
Comment (digitalchild) [inline templates/dashboard/orders.php:199]: refunded is misspelled.

**Review by digitalchild — CHANGES_REQUESTED** [review]
The qty isn't showing on the order for what product is refunded. For the No Tax, we should leave that out for now. 
Vendor_Dashboard_–_WC_Vendors

**Comment by Mrlunsuper** [comment]
> The qty isn't showing on the order for what product is refunded. For the No Tax, we should leave that out for now. Vendor_Dashboard_–_WC_Vendors

Can you show me the order details and product details?

**Comment by digitalchild** [comment]
Order details here. 
Edit_order_“Order_–_July_29__2021___05_21_AM”_‹_WC_Vendors_—_WordPress

**Comment by digitalchild** [comment]
Refund line should read as follows: 

**Refunded:** 1x Price - Product Name

**Comment by bdesai1982** [comment]
Steps to execute:
1. Customer places an order for multiple products in a single order 
2. Admin initiates a refund for any one product
3. The status in the commission's table is updated from due --> reversed only for the vendor but not for the admin
4. But the status is updated for both the admin and the vendor when the entire order is placed for a refund.
Screenshot_15

Screenshot_16

**Review by digitalchild — CHANGES_REQUESTED** [review]
The qty on the orders display is still showing 0. 
Vendor_Dashboard_–_WC_Vendors

**Review by digitalchild — APPROVED** [review]

**Pull request merged**